### PR TITLE
Feat/use w3m qrcode

### PR DESCRIPTION
**Pull request by devceline**
# Description

- Use  `W3mQrCode` instead of `qrcode` to display the QRCode
- Update `w3m` theme when updating `w3i` theme

# Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Draft PR (breaking/non-breaking change which needs more work for having a proper functionality _[Mark this PR as ready to review only when completely ready]_)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Examples/Screenshots

| Before       | After        |
| ------------ | ------------ |
| image | image |
| image  | image |

# Checklist:

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/account/Share/Share.tsx [hunk 1]
```diff
@@ -104,11 +80,7 @@ export const Share: React.FC = () => {
           </button>
         </div>
         <div className="Share__qr">
-          {qrCode ? (
-            <img src={`data:image/svg+xml;utf8,${encodeURIComponent(qrCode)}`} />
-          ) : (
-            'Loading...'
-          )}
+          <W3mQrCode size={250} imageUrl={ensAvatar ?? '/logo.png'} uri={uri} />
```
Comment (boidushya) [inline src/components/account/Share/Share.tsx:83]: We could have the width as 318 to prevent extra whitespace on the sides

Comment (boidushya) [reply src/components/account/Share/Share.tsx:83]: We also have to change styling of the container div to not explicitly provide 250px as the width 
image

**Review by boidushya — COMMENTED** [review]
LGTM apart from these two issues!

File: src/components/account/Share/Share.tsx (hunk 1 above)
Comment (devceline) [reply src/components/account/Share/Share.tsx]: The `svg` underneath has fixed dimensions

**Comment by boidushya** [comment]
LGTM!
🚢

**Review by boidushya — APPROVED** [review]

**Pull request merged**